src/vsock/README.md

```diff
@@ -0,0 +1,87 @@
+# vhost-user-vsock
+
+## Design
+
+TODO: This section should have a high-level design of the crate.
```

Comment:
Yes, it would be very useful to have a high level description, also to review the code, so when you can it is better to add it :-)